src/server/pfs/server/api_server.go

```diff
@@ -401,14 +401,22 @@ type WalkCommitSubvenanceRequest struct {
 	*pfs.WalkCommitSubvenanceRequest
 }
 
-// WalkCommitProvenance implements the protobuf pfs.WalkCommitProvenance RPC
-func (a *apiServer) WalkCommitProvenance(request *WalkCommitProvenanceRequest, srv pfs.API_WalkCommitProvenanceServer) error {
-	return errors.New("not implemented")
+func (a *apiServer) WalkCommitProvenanceTx(ctx context.Context, request *WalkCommitProvenanceRequest, tx *pachsql.Tx, srv pfs.API_WalkCommitProvenanceServer) error {
```

Comment:
Fwiw, I'm of the opinion that the order of parameters should generally be `ctx, tx, ...`.